.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
-* @MonicaOlejniczak @JakeLane @at-nathan @blackheart-atlassian @pancaspe87 @TimeRaider @heliam1 @hannahkmolloy @harshmalde
+* @MonicaOlejniczak @JakeLane @at-nathan @pancaspe87
 **/package.json
 yarn.lock
```

Comment:
Should we put the [console.log back to avoid the warnings below](https://github.com/atlassian-labs/compiled/pull/1193#discussion_r860519974)?

Reply:
I've suppressed the eslint warning instead which should stop the CI action prompt